src/views/perms/AssetPermission/AssetPermissionCreateUpdate.vue

```diff
@@ -49,14 +50,15 @@ export default {
       createSuccessNextRoute: { name: 'AssetPermissionDetail' },
       fieldsMeta: {
         users: {
+          type: 'userSelect',
+          component: UserSelect,
+          rules: [{
+            required: false
+          }],
           el: {
             value: [],
-            ajax: {
-              url: '/api/v1/users/users/?fields_size=mini',
-              transformOption: (item) => {
-                return { label: item.name + '(' + item.username + ')', value: item.id }
-              }
-            }
+            defaultPageSize: 300,
+            baseUrl: '/api/v1/users/users/?fields_size=small'
           }
         },
         user_groups: {
```

Comment:
Differences and Suggestions: 

No major changes detected.

Suggestions:

1. Add documentation to `UserSelect` if its purpose is not clear.
2. Consider refactoring the `all accounts list` into more reusable sections like "users", "groups" etc. to clean up logic related to fetching data. The current structure seems disjointed but it's hard to say why without understanding what each section does exactly.
3. If possible, consider moving some commonly used utility functions or classes out of their components files. For example, having `accountFormatter`, `assetSelect` could be considered a common need that needs to be handled somewhere in your system. This makes them easier to test and also maintain.